src/ensembl/prettier.config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   singleQuote: true,
   arrowParens: 'always',
-  trailingComma: "none"
+  trailingComma: "none",
```

Comment:
Use single quotes instead?

Reply:
I only added a comma on this line...

Reply:
Updated

Reply:
Thanks. Yes I know it isn't related to PR but thought we'll just change it here itself 😄